assets/css/style.css

```diff
@@ -126,3 +126,31 @@ a {
 #file-upload {
     display: none;
 }
+body { margin: 0; }
+
+table { width: 100%; box-sizing: border-box; box-shadow: inset 0px 0px 0px 1px rgba(0,0,0,0.1); font-family: 'Ubuntu'; /*border-collapse: collapse;*/ border-spacing: 0; background-color: purple; }
+
+th, td { border: 1px solid rgba(255,255,255,0.1); box-sizing: border-box; }
+
+th { text-transform: uppercase; font-size:10px; font-weight:700; padding: 10px 0; color: rgba(255,255,255,0.5); background-color: #292929; letter-spacing: 1px; }
+
+td { width: 14.285%; transition: all 0.3s; font-size: 10px; color: rgba(255,255,255,0.6); font-weight: 400; font-size: 10px; padding: 1.5% 1.5% 5%; vertical-align: initial; padding: 1.5% 0 ; height: 120px; }
+
```

Comment:
It's fine since there's no tables elsewhere, but we don't want to style tags directly, should use classes instead.